README.md

```diff
@@ -54,6 +54,7 @@ by the Helm v2 client instance. [Clean up](#clean-up-helm-v2-data) should only b
 
 - Helm v2 client installed on a system which manages releases on one to many clusters
 - Helm v3 client with `2to3` plugin installed on the same system
+- If `export KUBECONFIG` is used to point to the Kubernetes cluster, it must have only one `kubeconfig` file e.g. `export KUBECONFIG=cluster1_config` as plugin doesn't handle multiple `kubeconfig` files.
```

Comment:
Maybe it would read better as follows:
"Access to the cluster(s) that Helm v2 client is managing and which Helm v3 will manage after migration. This access is similar to `kubectl` access using [kubeconfig files](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/). Note: If using the `KUBECONFIG` environment value, the plugin only supports a single kubeconfig file and not a list of them."

What do you think?

Reply:
sounds good

Reply:
done